RELEASING.md

```diff
@@ -91,7 +91,13 @@ The following credentials are required for publishing (and automatically set in
 * `SONATYPE_USER` and `SONATYPE_KEY`: Sonatype username and password.
```

Comment:
Let's also go ahead and describe / link to getting these for a maintainer

https://central.sonatype.org/publish/manage-permissions/